app/Mail/CommunityActivityMail.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mail;
+
+use App\Community\Enums\ArticleType;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class CommunityActivityMail extends Mailable
+{
+    use Queueable; use SerializesModels;
+
+    /**
+     * Create a new message instance.
+     */
+    public function __construct(
+        public string $toUserDisplayName,
+        public int $activityId,
+        public string $activityCommenterDisplayName,
+        public int $articleType,
+        public string $articleTitle,
+        public string $urlTarget,
+        public bool $isThreadInvolved = false,
+        public ?string $payload = null,
+    ) {
+    }
+
+    /**
+     * Get the message envelope.
+     */
+    public function envelope(): Envelope
+    {
+        return new Envelope(
+            subject: $this->buildEmailSubject(
+                $this->articleType,
+                $this->activityCommenterDisplayName
+            ),
+        );
+    }
+
+    /**
+     * Get the message content definition.
+     */
+    public function content(): Content
+    {
+        return new Content(
+            markdown: 'mail.community.activity',
+            with: [
+                'activityDescription' => $this->buildActivityDescription(
+                    $this->articleType,
+                    $this->articleTitle,
+                    $this->toUserDisplayName,
+                    $this->isThreadInvolved,
+                ),
+            ],
+        );
+    }
+
+    /**
+     * Get the attachments for the message.
+     *
+     * @return array<int, \Illuminate\Mail\Mailables\Attachment>
+     */
+    public function attachments(): array
+    {
+        return [];
+    }
+
+    private function buildActivityDescription(
+        int $articleType,
+        string $articleTitle,
+        string $toUserDisplayName,
+        bool $isThreadInvolved
+    ): string {
+        $activityDescription = $isThreadInvolved ? "a thread you've commented in" : "your latest activity";
+
+        switch ($articleType) {
+            case ArticleType::Game:
+                $activityDescription = "the game wall for {$articleTitle}";
+                break;
```

Comment:
Game wall comments are no longer displayed in the email.

Before:
![image](https://github.com/user-attachments/assets/e38dd4af-f53d-42c4-9d84-df780d4d8b0f)

After:
![image](https://github.com/user-attachments/assets/54392696-ba30-47ce-bfee-e6971ffb614e)

Reply:
Achievement comments too. I suspect all of these have a similar fate.

Reply:
I'm seeing something different locally:
![Screenshot 2025-03-23 at 3 46 33 PM](https://github.com/user-attachments/assets/de721143-a2e1-4777-853d-9d9919eab7a7)

How are you previewing these? When I switch to this branch:
* I have `sail up` and `sail artisan horizon` running in two separate terminal processes.
* I'm navigating to Mailpit at http://localhost:64050/ to view the emails.

Reply:
Seems fixed now. I run `sail up -d` and have horizon running in a separate terminal. I also use mailpit to view the emails.

Reply:
Ah, I remember us discussing this logic when the payload content was added to emails, and now I'm actually wondering if this is functioning as designed.

New users _shouldn't_ have their content attached in the emails, as this prevents people from weaponizing our email system.